client/dcr: Harness test `TestWallet` fails.

In `client/asset/dcr`
```
$ go test --tags harness

.....

2023-04-07 18:57:10.934 [DBG] TEST: tip change: 608 (686a14844f82172fcf70f2d89659f3e0a536638a35e0adb9b9cab8fa1164e6d6) => 609 (59a3e66e6eb86eed773a47c1a350cc434eb815d9ca6b1992f76b229de62a7ae1)
2023-04-07 18:57:10.935 [INF] TEST: alpha has reported a new block, error = <nil>
2023-04-07 18:57:10.935 [DBG] TEST: tip change: 608 (686a14844f82172fcf70f2d89659f3e0a536638a35e0adb9b9cab8fa1164e6d6) => 609 (59a3e66e6eb86eed773a47c1a350cc434eb815d9ca6b1992f76b229de62a7ae1)
2023-04-07 18:57:10.935 [INF] TEST: beta has reported a new block, error = <nil>
2023-04-07 18:57:11.976 [TRC] TEST: Output d00797901fcd39a1a9ff03039a3ae6428beb615f377b016f304c7b7cbd1ff6c9:0 was not reported as unspent by SPV wallet. Spend status UNKNOWN.
2023-04-07 18:57:11.976 [INF] TEST: Output d00797901fcd39a1a9ff03039a3ae6428beb615f377b016f304c7b7cbd1ff6c9:0 NOT yet found; now searching with block filters.
2023-04-07 18:57:11.976 [DBG] TEST: Searching for tx d00797901fcd39a1a9ff03039a3ae6428beb615f377b016f304c7b7cbd1ff6c9 in blocks between best block 609 (59a3e66e6eb86eed773a47c1a350cc434eb815d9ca6b1992f76b229de62a7ae1) and the block just before 2023-04-07 18:57:08.931177359 +0900 JST m=+3.134328512.
2023-04-07 18:57:11.978 [DBG] TEST: Found mined tx d00797901fcd39a1a9ff03039a3ae6428beb615f377b016f304c7b7cbd1ff6c9 in block 59a3e66e6eb86eed773a47c1a350cc434eb815d9ca6b1992f76b229de62a7ae1.
--- FAIL: TestWallet (3.05s)
    simnet_test.go:79: peer count = 3, err = <nil>
    simnet_test.go:79: peer count = 1, err = <nil>
    simnet_test.go:468: checkConfs: expected spent = false, got true
FAIL
exit status 1
FAIL    decred.org/dcrdex/client/asset/dcr      9.183s
```

--- a/client/asset/dcr/simnet_test.go
+++ b/client/asset/dcr/simnet_test.go
@@ -464,8 +464,7 @@ func runTest(t *testing.T, splitTx bool) {
 	mineAlpha()
 	waitNetwork()
 	// Check that the swap has one confirmation.
-	expectSpent := !betaSPV // spv wallets SwapConfirmations will never report their own swap as spent
-	checkConfs(1, expectSpent)
+	checkConfs(1, true)
 	if !blockReported {
 		t.Fatalf("no block reported")
 	}

